### PR TITLE
Robin/pie

### PR DESCRIPTION
**Pull request by rjkerrison**
Something to test the water of creating graphics from data using custom SVG.

Has the potential to be a little lighter weight than using d3, and I think it should allow us to use Contentful to define the data, provided we can create a few template graphics.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    },
+  },
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|md)$':
+      '<rootDir>/config/assets-transformer.ts',
+    '\\.(css|less|scss)$': '<rootDir>/config/assets-transformer.ts',
+  },
+  setupFilesAfterEnv: ['<rootDir>/config/enzyme.ts', '<rootDir>/config/env.ts'],
+  snapshotSerializers: ['enzyme-to-json/serializer'],
+  testMatch: ['**/*.(test|spec).(ts|tsx)'],
```
Comment (tkeburia) [inline jest.config.js:18]: I suppose until we rewrite all `js` files to typescript, it's useful to match both, same for the transform matcher below

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
```
Comment (tkeburia) [inline jest.config.js:1]: Since I've added this file as well, there's gonna be some conflicts - sorry 🙊

**Review by tkeburia — COMMENTED** [review]
Hm, the build keeps failing for me. First it was complaining about missing the `@babel/plugin-proposal-class-properties` plugin. After I added that I'm still getting webpack errors. Maybe there's some dependency you have in your global `node_modules` that you need and didn't include here?

**Comment by rjkerrison** [comment]
the biggest part of this was taken into #46, so closing this as unnecessary now

**Pull request closed without merging**